hawkbit-repository/src/main/java/org/eclipse/hawkbit/MultiTenantJpaTransactionManager.java

```diff
@@ -38,38 +37,31 @@
     protected void doBegin(final Object transaction, final TransactionDefinition definition) {
         super.doBegin(transaction, definition);
 
+        // ignore transactions on tenant independent calls
```

Comment:
For this comment you can write a method, which explains the code e.g. `isTenantIndependent`